tests/brevitas/quant_tensor/test_quant_tensor.py

```diff
@@ -100,7 +100,7 @@ def test_quant_tensor_view():
 def test_is_valid():
     x = torch.randn(4, 4)
     # directly initialised QuantTensor shouldn't be valid
-    invalid_quant_tensor = QuantTensor(x)
+    invalid_quant_tensor = QuantTensor(x, None, None, None, None, None)
```

Comment:
It is actually possible to initialize a valid QuantTensor from scratch, if you handpick your values, so the comment is not precise.

Instead of picking None which causes only pain, i would pick random number.

Reply:
I added None because some of the test workflows weren't accepting the default arguments.
The comment is just referring to the scenario where its directly initiated without doing anything else but I guess it is ambiguous.